semantic-conventions/README.md

```diff
@@ -2,6 +2,23 @@
 
 A docker image to process Semantic Convention YAML models.
 
+## Development
+
+**Build Dependencies**:
+
+- pip
+- docker
+
+**Build**:
+Run the scripts described in `.github/workflows/semconvgen.yml`
+
+Additionally, the following python modules need to be on the path
+
+```
+pip install -r dev-dependencies.txt
```

Comment:
This is already in the yml, so it's not really "additionally". Still good to mention it.

Reply:
+1 in adding the actual steps here or in a CONTRIBUTING.md.